stock_cost_method_last/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Last Price Costing Method',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse',
+    'license': 'AGPL-3',
+    'summary': 'Add a new Costing Method *Last Price*',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```

Comment:
@alexis-via URL should be https://github.com/OCA/stock-logistics-workflow

Reply:
@Nikul-Chaudhary done